diff --git a/alts/BUILD.bazel b/alts/BUILD.bazel
index 819daedcc..96e826a8e 100644
--- a/alts/BUILD.bazel
+++ b/alts/BUILD.bazel
@@ -22,6 +22,7 @@ java_library(
         artifact("com.google.j2objc:j2objc-annotations"),
         artifact("io.netty:netty-buffer"),
         artifact("io.netty:netty-codec"),
+        artifact("io.netty:netty-codec-http"),
         artifact("io.netty:netty-common"),
         artifact("io.netty:netty-handler"),
         artifact("io.netty:netty-transport"),

